packages/svelte-ux/src/routes/docs/utils/format/+page.svelte

```diff
@@ -83,13 +102,277 @@
+        custom: 'eee, MMMM do',
+      })}
+    </Preview>
+  </div>
+  <div>
+    <h3>With descriptive tokens</h3>
+    <Preview>
+      {format(myDate, PeriodType.Custom, {
+        custom: [DateToken.DayOfWeek_short, DateToken.Month_long, DateToken.DayOfMonth_withOrdinal],
+      })}
+    </Preview>
+  </div>
+  <div>
+    <h3>With full intl</h3>
+    <Preview>
+      {format(myDate, PeriodType.Custom, {
+        custom: { weekday: 'short', month: 'long', day: 'numeric', withOrdinal: true },
+      })}
+    </Preview>
+  </div>
+</div>
+
+<h2>PeriodType Day</h2>
+
+<div class="grid grid-cols-3 gap-4">
+  <div>
+    <h3>short</h3>
+    <Preview>
+      {format(myDate, PeriodType.Day, {
+        variant: 'short',
+      })}
+    </Preview>
+  </div>
+  <div>
+    <h3>default</h3>
+    <!-- TODO JYC QUESTION: before: 3/30/1982, suggestion as default: 03/30/1982 -->
+    <Preview>
+      {format(myDate, PeriodType.Day, {
+        // variant: 'default',
+      })}
+    </Preview>
+  </div>
+  <div>
+    <h3>long</h3>
+    <Preview>
+      {format(myDate, PeriodType.Day, {
+        variant: 'long',
+      })}
+    </Preview>
+  </div>
+</div>
+
+<h2>PeriodType Week</h2>
+<span>
+  It will take your default <b>weekStartsOn</b>
+  <a class="text-accent-500" href="/customization#settings">settings</a>, if you want to be
+  specific, you can also use
+  <b>PeriodType.WeekSun</b>
+</span>
+<div class="grid grid-cols-3 gap-4">
+  <div>
+    <h3>short</h3>
+    <Preview>
+      {format(myDate, PeriodType.Week, {
+        variant: 'short',
+      })}
+    </Preview>
+  </div>
+  <div>
+    <h3>default</h3>
+    <Preview>
+      {format(myDate, PeriodType.Week, {
+        // variant: 'default',
+      })}
+    </Preview>
+  </div>
+  <div>
+    <h3>long</h3>
+    <Preview>
+      {format(myDate, PeriodType.Week, {
+        variant: 'long',
+      })}
+    </Preview>
+  </div>
+</div>
+
+<h2>PeriodType BiWeek1</h2>
+<span>
+  It will take your default <b>weekStartsOn</b>
+  <a class="text-accent-500" href="/customization#settings">settings</a>, if you want to be
+  specific, you can also use
+  <b>PeriodType.BiWeek1Sun</b>
+</span>
+<div class="grid grid-cols-3 gap-4">
+  <div>
+    <h3>short</h3>
+    <Preview>
+      {format(myDate, PeriodType.BiWeek1, {
+        variant: 'short',
+      })}
+    </Preview>
+  </div>
+  <div>
+    <h3>default</h3>
+    <Preview>
+      {format(myDate, PeriodType.BiWeek1, {
+        // variant: 'default',
+      })}
+    </Preview>
+  </div>
+  <div>
+    <h3>long</h3>
+    <Preview>
+      {format(myDate, PeriodType.BiWeek1, {
+        variant: 'long',
+      })}
+    </Preview>
+  </div>
+</div>
+
+<h2>PeriodType Month</h2>
+
+<div class="grid grid-cols-3 gap-4">
+  <div>
+    <h3>short</h3>
+    <!-- TODO JYC QUESTION: before: Mar, suggestion as default: 3 -->
+    <Preview>
+      {format(myDate, PeriodType.Month, {
+        variant: 'short',
+      })}
+    </Preview>
+  </div>
```

Comment:
What do you think about this change?
Just a question to have a bit better default default. Let me know

Reply:
I think it should still be "Mar" for short and "March" for long.  If only showing the month by itself, only showing a number I think can be confusing, or am I overlooking a use case?

Reply:
We have 3 levels now, today it's: 
```ts
{
  'short': "Mar"
  'default': "Mar"
  'long': "March"
}
```

That's why I was wondering to have another default default for `short` as `Mar` is already covered in `default`.

Reply:
Awe, good point... hmm.  It is odd to have 2 result in the same.  I'm ok with `default` returning `3` in this case then.  Thanks